tests/ci/build_run_benchmarks.sh

```diff
@@ -7,49 +7,46 @@ set -x
 # set default value of directory name
 if [ -z "${PR_FOLDER_NAME}" ]; then export PR_FOLDER_NAME=aws-lc; fi
 
+# Get AWS_ACCOUNT_ID
+if [ -z "${AWS_ACCOUNT_ID}" ]; then AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text); fi
+
+AWSLC_PR_ROOT=$(pwd)
+
+cd ..
+
 # run this from the bm_framework root directory!
-OPENSSL_ROOT=$(pwd)/openssl
-BORINGSSL_ROOT=$(pwd)/boringssl
 AWSLC_PR_ROOT=$(pwd)/"${PR_FOLDER_NAME}"
 AWSLC_PROD_ROOT=$(pwd)/aws-lc-prod
 
+NUM_CPU_THREADS=$(nproc)
```

Comment:
NP: Can this script depend on common_posix_setup? It has a smarter NUM_CPU_THREADS variable, I know not all systems have nproc by default, but we probably wont run the benchmark on those systems anyway.